parametric/test_tracer.py

```diff
@@ -61,7 +61,7 @@ def test_tracer_service_name_environment_variable(
     assert span["service"] == library_env["DD_SERVICE"]
 
 
-@parametrize("library_env", [{"DD_ENV": "prod"}, {"DD_ENV": "dev"}])
```

Comment:
Why the removal of dev?

Reply:
there's no real benefit here to testing both, we don't expect any different outcome other than the tag changing which is sufficiently tested by just using one value